tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
```

Comment:
We probably want `declarations: true` here so that typescript will generate type information for anyone using this module.